### PR TITLE
Block translation

### PR DESCRIPTION
**Pull request by core23**

### CODE REVIEW

File: Resources/translations/SonataFormatterBundle.cs.xliff [hunk 1]
```diff
@@ -6,6 +6,10 @@
                 <source>please_select_format_method</source>
                 <target>vyberte možnost formátování textu</target>
             </trans-unit>
+            <trans-unit id="sonata_formatter_block_formatter">
+                <source>sonata.formatter.block.formatter</source>
+                <target>Rich Text Area</target>
```
Comment (OskarStark) [inline Resources/translations/SonataFormatterBundle.cs.xliff]: if you don't have the correct translation, please use the ``source`` as ``target`` and NOT the english one.

thank you

**Comment by core23** [comment]
@Soullivaneuh Can you help me with the French translation?

**Comment by OskarStark** [comment]
@rande can translate it too :smile:

**Comment by rande** [comment]
sonata.formatter.block.formatter => Texte Riche

**Comment by core23** [comment]
Thanks @rande

**Comment by OskarStark** [comment]
ping @sagikazarmark :smile:

**Comment by sagikazarmark** [comment]
``` xml
<trans-unit id="sonata_formatter_block_formatter">
    <source>sonata.formatter.block.formatter</source>
    <target>Rich Text</target>
</trans-unit>
```

There is no expression for this in Hungarian.

**Comment by core23** [comment]
Thank you @sagikazarmark

**Comment by OskarStark** [comment]
Thank you @core23 , awesome work, like ever... :smile:

**Pull request merged**